gui/src/2D/controls/inputText.ts

```diff
@@ -289,6 +338,23 @@ export class InputText extends Control implements IFocusableControl {
 
     /** @hidden */
     public processKey(keyCode: number, key?: string, evt?: KeyboardEvent) {
+
+        //get the clipboard Event from the keycode, returns -1 if not found.
+        let clipboardEventType: ClipboardEventTypes = ClipboardInfo.GetTypeFromCharacter(keyCode);
```

Comment:
This should not be here but more inside the observer created by the input text to subscribe to ADT.onClipboardEvent